### ISSUE TITLE
server.createList does not use factory unless seeds() is used

### ISSUE THREAD

**Issue by IanVS** [miragejs/miragejs#916]
This one was a bit confusing for me to figure out.  I defined a single model (named 'host'), and a factory for the model, but I found that the factory wasn't used when I did this:

```
const server = createServer(serverConfig);
server.createList('host', 5);
```

In that case, I got an array of five objects with only an `id`.  BUT, if I do something like this:

```
const server = createServer({...serverConfig, seeds() {/* nothing here */}});
server.createList('host', 5);
```

Then the factory kicks in and works correctly.

**Comment by samselikoff** [comment]
yeah. Sadly hangover from an old/bad api where we either defaulted to fixtures or factories. do you have fixtures defined?

**Comment by IanVS** [comment]
No, I only have `models`, `factories`, `serializers`, and `routes`.
